src/components/templates/Jazzbar/components/ReactionList.tsx

```diff
@@ -52,6 +49,7 @@ const ReactionList: React.FC<ReactionListProps> = ({
   const profileImageSize = small
     ? REACTION_PROFILE_IMAGE_SIZE_SMALL
     : REACTION_PROFILE_IMAGE_SIZE_LARGE;
+
   return (
     <>
       <div className={`reaction-list ${small && "small"}`}>
```

Comment:
In all cases where there is only going to be a single root `<div>`/etc element now that we've removed the modal, we'll no longer need to wrap it with this fragment (`<>`)